tests/test_pyglmnet.py

```diff
@@ -388,3 +385,61 @@ def test_random_state_consistency():
 
     assert_array_equal(ypred_a, ypred_b)
     assert_array_equal(ypred_b, ypred_c)
+
+
+@pytest.mark.parametrize("distr", ALLOWED_DISTRS)
+def test_simulate_glm(distr):
+    """
+    Test that every generative model can be simulated from.
+    """
+
+    n_samples, n_features = 10, 3
+
+    # sample random coefficients
+    beta0 = np.random.normal(0.0, 1.0, 1)
+    beta = np.random.normal(0.0, 1.0, n_features)
+
+    np.random.seed(1)
+    X = np.random.normal(0.0, 1.0, [n_samples, n_features])
+    simulate_glm(distr, beta0, beta, X)
```

Comment:
this is just a smoke test. Is there a benefit to having it separately than what already exists?